Dockerfile

```diff
@@ -54,7 +57,9 @@ RUN curl -O ftp://ftp.unidata.ucar.edu/pub/netcdf/netcdf-4.4.1.1.tar.gz \
     && /bin/sh configure --prefix=/usr/local \
     && make \
     && make install \
-    && cd ..
+    && cd .. \
+    && rm -rf netcdf-4.4.1.1 \
+    && rm -rf netcdf-4.4.1.1.tar.gz
 
 RUN yum -y install httpd
 RUN yum clean all
```

Comment:
This step won't actually reduce the size of the build despite removing old package metadata, as previous build steps will still have the metadata.  It'd be good to combine these lines into the `RUN` command with all the `&&` chains so that the space is actually freed for the image.